README.rst

```diff
@@ -301,6 +301,25 @@ This works for custom MFEs, as well. For example, if you added your own MFE name
 
 Similarly, in production, the "mfe" Docker image will be rebuilt automatically during ``tutor local launch``.
 
+Note: the name of the bind-mount folder needs to match the name of the repository exactly word-for-word. If you've forked an MFE repository with a custom name, be sure to change the name back to ensure the bind-mount works properly.
+
+Using shared components for local development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Developers may want to use local repositories of common components such as `frontend-component-footer <https://github.com/openedx/frontend-component-footer>`_ and `frontend-component-header <https://github.com/openedx/frontend-component-header>`_. In the following example, let's assume one has cloned the example MFE and the shared components into a directory called ``workspace``. In the root of the example MFE, create a file called ``module.config.js`` and use the following to ensure your example MFE picks up the locally hosted components during build time:
```

Comment:
My understanding is that this paragraph only partially explains how to work on shared components. How is the "frontend-component-header" bind-mounted in the container at runtime? And at build-time? How did you get it to work _exactly_?

I think that your explanations would be valuable to other users, but we need to provide comprehensive instructions.

Reply:
Hi @regisb, you're right - I'm having trouble reproducing the steps even though I had gotten it to work at some point. I'll update the PR to omit the shared components bit and I'll add another PR in the future when I figure out how to get it working again.